src/test/java/com/bloomtech/asylumrgbea/services/impl/AsylumCaseServiceImplTest.java

```diff
@@ -1,100 +1,100 @@
-                ("SAN", "HIO", "Other", "Pending", "N/A",
-                        "6/24/2022");
-
-        Page<AsylumCase> pageEntities = new PageImpl<>(List.of(asylumCase));
-
-        PageResponseDto pageResponseDto = new PageResponseDto(1, List.of(asylumCaseResponseDto));
-
-        // WHEN
-        when(asylumCaseRepository.findAll(PageRequest.of(asylumCaseRequestDto.getPageNumber(), asylumCaseRequestDto.getNumberOfItemsInPage()))).thenReturn(pageEntities);
-        when(asylumCaseMapper.pagesToResponseDtos(pageEntities)).thenReturn(List.of(asylumCaseResponseDto));
-        when(asylumCaseMapper.pageDataAndPageToResponseDto(pageEntities.getTotalPages(), List.of(asylumCaseResponseDto))).thenReturn(pageResponseDto);
-
-        PageResponseDto result = asylumCaseService.getPageOfAsylumCases(asylumCaseRequestDto);
-
-        // THEN
-        assertNotNull(result);
-        assertEquals(1, result.getTotalPages());
-        assertTrue(result.getPage().iterator().hasNext());
-        verify(asylumCaseRepository).findAll(any(Pageable.class));
-        verify(asylumCaseMapper).pageDataAndPageToResponseDto(anyInt(), anyIterable());
-
-    }
-}
+//package com.bloomtech.asylumrgbea.services.impl;
+//
+//import com.bloomtech.asylumrgbea.controllers.exceptions.AsylumCaseNotFoundException;
+//import com.bloomtech.asylumrgbea.entities.AsylumCase;
+//import com.bloomtech.asylumrgbea.mappers.AsylumCaseMapper;
+//import com.bloomtech.asylumrgbea.models.AsylumCaseRequestDto;
+//import com.bloomtech.asylumrgbea.models.AsylumCaseResponseDto;
+//import com.bloomtech.asylumrgbea.models.PageResponseDto;
+//import com.bloomtech.asylumrgbea.repositories.AsylumCaseRepository;
+//import org.junit.jupiter.api.BeforeEach;
+//import org.junit.jupiter.api.Test;
+//import org.mockito.Mockito;
+//import org.springframework.data.domain.Page;
+//import org.springframework.data.domain.PageImpl;
+//import org.springframework.data.domain.PageRequest;
+//import org.springframework.data.domain.Pageable;
+//
+//import java.util.Collections;
+//import java.util.List;
+//
+//import static org.junit.jupiter.api.Assertions.*;
+//import static org.mockito.Mockito.*;
+//
+//class AsylumCaseServiceImplTest {
+//    private AsylumCaseMapper asylumCaseMapper;
+//    private AsylumCaseRepository asylumCaseRepository;
+//    private AsylumCaseServiceImpl asylumCaseService;
+//
+//    @BeforeEach
+//    void setup() {
+//        asylumCaseMapper = mock(AsylumCaseMapper.class);
+//        asylumCaseRepository = mock(AsylumCaseRepository.class);
+//        asylumCaseService = new AsylumCaseServiceImpl(asylumCaseRepository, asylumCaseMapper);
+//    }
+//
+//    @Test
+//    void getAllAsylumCases_givenValidAsylumCase_returnsIterableOfAsylumCaseResponseDto() {
+//        // GIVEN
+//        AsylumCase asylumCase = new AsylumCase();
+//        asylumCase.setId("aaa345");
+//        AsylumCaseResponseDto asylumCaseResponseDto = new AsylumCaseResponseDto(
+//                "SAN", "HIO", "Other",
```

Comment:
To better facilitate familiarizing new team members with the proper structure of our case data, it may be best to use test data that mimics what actual data (an instance of Asylum Case) might contain. While this is no way changes the outcome from an application standpoint, it can help the people who write it.

Reply:
I completely agree. 

This is existing test data that needs to updated. We will implement functional tests that mimic the actual data in the next ticket